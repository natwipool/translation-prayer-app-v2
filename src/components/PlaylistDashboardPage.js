import React from 'react';
import { Link } from 'react-router-dom';
import PlaylistList from './PlaylistList';

const PlaylistDashboardPage = () => (
  <div>
    <Link to="/create"> 
      สร้างรายการสวดมนต์
    </Link>
    <PlaylistList />
  </div>
)

export default PlaylistDashboardPage;