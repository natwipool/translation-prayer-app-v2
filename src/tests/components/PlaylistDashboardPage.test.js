import React from 'react';
import { shallow } from 'enzyme';
import PlaylistDashboardPage from '../../components/PlaylistDashboardPage';

test('should render PlaylistDashboardPage correctly', () => {
  const wrapper = shallow(<PlaylistDashboardPage />)
  expect(wrapper).toMatchSnapshot();
});