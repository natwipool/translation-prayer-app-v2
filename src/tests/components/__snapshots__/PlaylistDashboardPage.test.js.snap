// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PlaylistDashboardPage correctly 1`] = `
<div>
  <Link
    replace={false}
    to="/create"
  >
    สร้างรายการสวดมนต์
  </Link>
  <Connect(PlaylistList) />
</div>
`;
